gax-java/gax/src/test/java/com/google/api/gax/core/GaxPropertiesTest.java

```diff
@@ -57,10 +57,85 @@ public void testGaxVersion() {
     }
   }
 
+  private static String originalJavaVersion = System.getProperty("java.version");
+  private static String originalJavaVendor = System.getProperty("java.vendor");
+  private static String originalJavaVendorVersion = System.getProperty("java.vendor.version");
+
+  public static void cleanup() {
```

Comment:
Use `@After` https://junit.org/junit4/javadoc/4.13/org/junit/After.html to avoid calling `cleanup()`.